api/config/config_preprod.json

```diff
@@ -148,5 +148,9 @@
   },
   "app": {
     "rStudioDescriptorPath": "https:\/\/raw.githubusercontent.com\/DataBiosphere\/terra-app\/39c602e20ba027eb065dcb7690e76f2236ac2848\/apps\/rstudio\/app.yaml"
+  },
+  "termsOfService": {
+    "latestAouVersion": 1,
+    "latestTerraTosTimestamp": "2030-12-31T12:34:56Z"
```

Comment:
Staging, Stable, Preprod, Prod: an arbitrary time in the distant **future** which we will soon replace with the actual value.

My plan is to wait until after Terra ToS officially launches in Terra-Prod, and update the value based on the events of that day.  This might mean a config-only push to these envs.